content/learning-paths/11111111-1111-1111-1111-111111111111/mastering-kubernetes-for-engineers/ambassador-edge-stack-api-gateway-with-meshery/conclusion/quiz.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Quiz"
+passing_percentage: 70
+layout: "test"
```

Comment:
Perhaps, it helps or maybe it doesn't hurt, but do we need to specify layout: "test"?

Reply:
Neither helps, nor hurts, it seems.